util/dnsname/dnsname.go

```diff
@@ -20,14 +20,12 @@ const (
 // A FQDN is a fully-qualified DNS name or name suffix.
 type FQDN string
 
-func ToFQDN(s string) (FQDN, error) {
+// NewFQDN converts a string to a FQDN that retains any leading '.' in the case of wildcards.
```

Comment:
This package should have no concept of wildcards or our internal/wire representation of them.